public/index.html

```diff
@@ -81,6 +81,51 @@
             content="https://i.imgur.com/JDAyCFO.png"
         />
         <meta property="og:image" content="https://i.imgur.com/JDAyCFO.png" />
+        <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
All of this seems to be duplicated in tailwind.config.ts, can it only reside in one place?